File: Deliver/Models/Db/Delivery.cs
using System.ComponentModel.DataAnnotations.Schema;

namespace Models.Db;

[Table("Deliveries")]
public class Delivery : BaseHashModel
{
    public string Name { get; set; }
    public DateTime StartDate { get; set; }
    public DateTime EndDate { get; set; }
    public int Status { get; set; }
    
    public long? CarId { get; set; }
    public Car Car { get; set; }

    public long FromId { get; set; }
    public Location From { get; set; }

    public long ToId { get; set; }
    public Location To { get; set; }
}

File: jsApps/launcher/src/service/deliveryService/models/Delivery.ts
import Car from "service/carService/models/Car";
import Location from "service/location/models/Location";

export default interface Delivery {
    hash: string;
    name: string;
    status: number;
    startDate: Date;
    endDate: Date;
    from: Location;
    to: Location;
    car?: Car;
}
